modules/ROOT/pages/troubleshoot-visualizer.adoc

```diff
@@ -39,10 +39,10 @@ If you encounter an error or issue while using Visualizer, you can try these sol
 |`You don’t have permissions to edit this node. Learn more.`
 |You need to have the *Visualizer Editor* permission to edit in Visualizer.
 |Learn more about the different permissions in Visualizer and how to configure xref:access-management::roles.adoc#custom-roles[custom roles].
-|`Metrics are hidden for this app because you don’t have the *Runtime Manager >> Read Applications* permission for its environment. Learn more.`
-|You have selected an application that belongs to an environment for which you do not have the *Read Applications* permission. The node is showing only because it is a dependency of another node.
+|`Metrics are hidden for this app because you don’t have the *Runtime Manager "Read Application"* permission for its environment. Learn more.`
```

Comment:
I wouldn't mark Runtime Manager in bold, would just mark Read Application in bold since that's the UI bit they'd be looking for.

Reply:
Quotes don't mean anything in particular...

Reply:
I only put it in quotes since it's in quotes in our UI and I wanted the messages to match.